packages/just-scripts/src/monorepo/getStackDiffs.ts

```diff
@@ -0,0 +1,47 @@
+import fs from 'fs-extra';
+import glob from 'glob';
+import { downloadPackage, exec, paths, encodeArgs } from 'just-scripts-utils';
+import path from 'path';
+import { DiffInfo } from './DiffInfo';
+import { diff_match_patch, patch_obj, Diff } from 'diff-match-patch';
```

Comment:
This is silly but can we rename these imports to look like JS things not python or whatever? :P

Reply:
Sigh, agreed. I can add aliases.